Sources/TokamakDOM/DOMNode.swift

```diff
@@ -54,7 +54,13 @@ extension _AnimationBoxBase._Resolved._RepeatStyle {
 }
 
 extension AnyHTML {
-  func update(dom: DOMNode, transaction: Transaction) {
+  func update(
+    dom: DOMNode,
+    additionalAttributes: [HTMLAttribute: String],
+    transaction: Transaction
+  ) {
```

Comment:
Would it be possible to split the body of this function into multiple functions or some helper types to resolve the linter warning?